vNext/src/OrleansRuntime/error.txt

```diff
@@ -0,0 +1,11 @@
+The process cannot access the file 'D:\microsoft\orleans-xiazen\vNext\src\OrleansRuntime\obj\Debug\netstandard1.5\Generated\OrleansRuntime.codegen.cs' because it is being used by another process.
```

Comment:
will do , sign I must be really messy last Wednesday. This is not good to include in !